cmd/minikube/cmd/config/set.go

```diff
@@ -30,13 +30,17 @@ var configSetCmd = &cobra.Command{
 	Use:   "set PROPERTY_NAME PROPERTY_VALUE",
 	Short: "Sets an individual value in a minikube config file",
 	Long: `Sets the PROPERTY_NAME config value to PROPERTY_VALUE
-	These values can be overwritten by flags or environment variables at runtime.`,
+Config values are global and will be applied to all profiles.
+These values can be overwritten by flags or environment variables at runtime.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		if len(args) < 2 {
 			exit.Message(reason.Usage, "not enough arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE", out.V{"ArgCount": len(args)})
 		}
 		if len(args) > 2 {
-			exit.Message(reason.Usage, "toom any arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE", out.V{"ArgCount": len(args)})
+			exit.Message(reason.Usage, "too many arguments ({{.ArgCount}}).\nusage: minikube config set PROPERTY_NAME PROPERTY_VALUE", out.V{"ArgCount": len(args)})
+		}
+		if cmd.Flags().Lookup(config.ProfileName).Changed {
+			exit.Message(reason.Usage, "Persistent config is global. Setting a specific profile name is not allowed.")
```

Comment:
how about exiting, we just 
```
out.BoxedErr("Persistent config is global. Setting a specific profile name for global config is  ignored")
```
I assume many automation might have a "-p" passed to the minikube for simplicity, we should not break existing workflows but only warn them that this -p is useless.

Reply:
Thanks for the feedback! Changed to `BoxedErr` in 4ed590048c54e3f49442f8074af52a3473dfa62f

![Screen Shot 2021-05-19 at 2 20 02 AM](https://user-images.githubusercontent.com/1311594/118765271-d673c700-b848-11eb-8521-3dbf13e77617.png)